rhino/README.md

```diff
@@ -0,0 +1,112 @@
+# Introduction
+
+Rhino supports two editors: IronPython and CPython. The following tutorials explain how to install and use compas on both.
```

Comment:
This starts off by saying something that applies only to Rhino 8 and then the first part is about Rhino 7. I think this should be either a fully rhino 8 tutorial that shows how to install the cpython and the ironpython support, or have two tutorials one for 7 and one for 8.

Reply:
I removed this description to have 2 separation sections for 7 and 8.